lib/src/booleans/boolean_valid.dart

```diff
@@ -0,0 +1,148 @@
+import 'package:turf/polygon_to_line.dart';
```

Comment:
this is beautiful, so we don't need any other PR for that one? This is already finished?